### DISCUSSION TITLE
--constructor-args

### DISCUSSION THREAD

**Discussion by kevin-j-kennedy** [foundry-rs/foundry#4966] [Help]
Hi there, I am trying to deploy a contract using the "forge create" command that passes the WETH address on my deployment network as a constructor argument.  See below for the relevant lines in Solidity:

    constructor(address _wethAddress) {
        WETH_ADDRESS = _wethAddress;
    }

What would be the correct format for --constructor-args?

Thanks a lot!

**Comment by allwin199** [comment]
@kevin-j-kennedy you can also deploy using `forge script`

https://book.getfoundry.sh/tutorials/solidity-scripting
